eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -60,6 +62,11 @@ object ChannelCodecs extends Logging {
     // field and don't support additional features which is why all bits are set to 0.
   )
 
+  /**
+   * byte-aligned boolean codec
+   */
+  val boolean: Codec[Boolean] = bool(8)
```

Comment:
Not sure about the name, maybe I should go with `bool8` or even `byteAligned(bool)`

Reply:
I think `bool8` makes sense since we have `uint16`, `uint32`, etc.

Reply:
Done with 06207d1.